testsuite/testcases/Cargo.toml

```diff
@@ -14,7 +14,7 @@ rust-version = { workspace = true }
 
 [dependencies]
 anyhow = { workspace = true }
-aptos = { workspace = true, features = ["fuzzing"] }
+movement = { workspace = true, features = ["fuzzing"] }
```

Comment:
This is a zombie file resurrected by merge. As the crate does not exist in the movement fork, could be removed.